lib/pluginmanager/commands/install_command.rb

```diff
@@ -0,0 +1,108 @@
+# encoding: utf-8
+require "pluginmanager/command"
+require "bootstrap/util/compress"
+require "fileutils"
+require "pluginmanager/sources/http"
+require "pluginmanager/sources/local"
+
+class LogStash::PluginManager::InstallCommand < LogStash::PluginManager::Command
```

Comment:
because I tried to extract some logic away from the `pluginmanager/install.rb`  in a way that things like utility, fetch, verification functions are more wrap here ... this works on a similar way as `pack_command.rb` or the `command.rb` interface in the main directory.